app/js/tabmanager.js

```diff
@@ -1,7 +1,10 @@
 /* @flow */
 /* global TW */
 
-import _ from 'underscore';
+// import _ from 'underscore';
```

Comment:
This doesn't seem correct. Were you testing replacing Underscore with Lodash? Lodash is not listed as a dependency.

Reply:
Ah, I nearly forgot about that. I was using lodash instead of underscore, because lodash caused an initialization error when running the tests. I couldn't find a solution to it, so I switched to lodash.
And although lodash isn't mentioned in the dependencies it's hoisted through some other dependencies. :-/
The question is if there is no solution to the initialization error with underscore, if we should make lodash a proper dependency and get rid of underscore. What do you think about that?

Reply:
Lodash is a second-level dependency only of another devDependency (babel uses it, for example). It's not used in released code, however. Using Lodash increases the .zip release file by 30KB.

I'm fine replacing Underscore with Lodash completely, but let's not ship both of them in the same .zip.

Reply:
Agree, we should only have either one of them only. I'm going to have another look at the underscore init issue. If I can't find a simple solution, I'm going to swap underscore out for lodash.

Reply:
This issue seems related to the testing problem: https://github.com/jashkenas/underscore/issues/2152

I'm not sure it's worth spending much time on this. If dropping in Lodash works, I'm cool with that. Feel free to go full Lodash and skip any more debugging. If you do, please use `yarn add` / `yarn remove` to update the lock file appropriately and replace `import _ from 'underscore';` everywhere with `import _ from 'lodash';`. TabWrangler's use of Underscore is  limited, so I problems from swapping should be limited.

Reply:
Yeah, that looks exactly what I was seeing. I've swapped underscore with lodash and updated the lock file.